### PR TITLE
Added paging using React Router

### PR DESCRIPTION
**Pull request by meedox9**
## Problem

We need a way to operate three independent applications from one source

## Solution

Added paging for the three app categories using React Router

Pages Added:

Menu -> /
Callup -> /callup
GarageSale -> /garage-sale
OpenHouse -> /open-house

## Testing

Tested locally, all url's check out and load the correct component

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,31 @@
 import React from "react"
 import './css/App.css';
-import Maps from "./Maps"
```
Comment (ghost) [inline src/App.js:3]: Assuming we will be making instances of these in subapplicatons

File: src/App.js [hunk 2]
```diff
@@ -1,12 +1,31 @@
...
+
 function App() {
   return (
     <div className="App">
-      {/*Map*/}
-      <Maps/>
+    <Router>
+      <Switch>
+        <Route  exact path="/" component={Menu} />
+        <Route  exact path="/callup" component={Callup} />
```
Comment (ghost) [inline src/App.js:21]: I suggest that you push these calls to the menu component.

**Review by ghost — CHANGES_REQUESTED** [review]
Minor changes should be made in src/App.js

**Comment by hennaceur** [comment]
Looks good, ready to merge!

File: src/App.js (hunk 2 above)
Comment (meedox9) [reply src/App.js:21]: I dont think so, it shouldn't matter, it is not rendering the components, it is just routing them :)

**Pull request merged**